rpcd/playbooks/maas-20151013-1-requirements.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Part of MaaS Migration: maas-20151013.yml
+- name: Ensure MaaS Migration Requirements
+  hosts: hosts
+  user: root
+  tasks:
+    - name: Ensure migrations index exists
+      local_action: file path={{ migrations_dir }} state=directory
+      when: inventory_hostname == groups['hosts'][0]
+    - name: Locate evidence of prior MaaS migration
+      local_action: stat path={{ migrations_dir }}/maas-20151013
+      when: inventory_hostname == groups['hosts'][0]
+      register: maas_upgraded_result
+      changed_when: False
+    - name: Stop if migraiton has been run
+      fail: msg="MaaS has been previously run migration 20151013141500"
+      when: maas_upgraded_result.stat.exists | bool
```

Comment:
This will fail on all nodes except hosts[0] because the "maas_upgraded_result" var was only created on host[0]. so we need the "when inventory_hostname == groups['hosts'][0]" clause I'm guessing.